plumber/api/importer.py

```diff
@@ -442,20 +585,40 @@ def import_mdl(
             file_system._fs,
             callbacks,
             threads,
+            # General settings
```

Comment:
Based on what was mentioned in the other comment, these should be named with a prefix related to what import type they belong to in the API surface